types/rrdtool/rrdtool-tests.ts

```diff
@@ -0,0 +1,138 @@
+import rrdtool, { DataSource, RoundRobinArchive, RrdtoolDatabase } from 'rrdtool';
```

Comment:
there's no default export in the shipped code, so you shouldn't assert that one will be available.

Reply:
amended